ddtrace/contrib/internal/aiobotocore/patch.py

```diff
@@ -45,6 +46,11 @@
     },
 )
 
+def _instrumented_versions() -> Dict[str, Collection[str]]:
+    return {
+        "aiobotocore": (">= 0.0.0", "< 1.0.0")
```

Comment:
```suggestion
        "aiobotocore": ">=0.0.0,<1.0.0"
```

`packaging.requirements.Requirement` can take what you would give `pip install`, so it can take a range like this, and it can also take markers, like `>=0.0.0,<1.0.0; python_version>="3.9"` if we wanted/needed to get specific for some.

your `Requirements(f"{name}{specifier}").contains(Version(...))` will still work with the `>=0.0.0,<1.0.0` specifier. It also means you can just do `<1.0.0` instead of needing the `>=0.0.0`.

Reply:
Makes sense to me, I've updated the file to use this spec.